stdlib/public/Concurrency/Task.swift

```diff
@@ -521,6 +521,45 @@ extension Task where Failure == Never {
 #endif
 }
 
+@available(SwiftStdlib 5.1, *)
+@_silgen_name("swift_task_enqueueTaskOnExecutor")
+func _enqueueTaskOnExecutor(_ task: Builtin.NativeObject, _ executor: UnownedSerialExecutor)
+
+@available(SwiftStdlib 5.1, *)
+extension Actor {
+  public nonisolated func send(
+    @_inheritActorContext @_implicitSelfCapture operation: __owned @Sendable @escaping (isolated Self) async -> Void
+  ) {
+    let flags = taskCreateFlags(
+            priority: nil, isChildTask: false, copyTaskLocals: false,
```

Comment:
```suggestion
            priority: nil, isChildTask: false, copyTaskLocals: true,
```

we SHOULD copy locals here actually